deepwell/src/methods/user.rs

```diff
@@ -45,24 +49,28 @@ pub async fn user_put(mut req: ApiRequest) -> ApiResponse {
     let reference = ItemReference::try_from(&req)?;
     let user = req.user(&txn).update(reference, input).await.to_api()?;
     txn.commit().await?;
-    build_user_response(&user, StatusCode::Created)
+    build_user_response(&user, UserDetails::default(), StatusCode::Created)
 }
 
 pub async fn user_delete(req: ApiRequest) -> ApiResponse {
     let txn = req.database().begin().await?;
     let reference = ItemReference::try_from(&req)?;
     let user = req.user(&txn).delete(reference).await.to_api()?;
     txn.commit().await?;
-    build_user_response(&user, StatusCode::Ok)
+    build_user_response(&user, UserDetails::default(), StatusCode::Ok)
```

Comment:
Are these methods intended to always be internal or will we eventually make these external and match the spec? I didn't bother adding query parameters because in the spec these methods wouldn't be returning anything other than a response code.

Reply:
Hmm, I suppose. We can change it later but I figured returning model data is something I can do, might make use of later, so why not.

I'm mostly suggesting this too because "why not", the generic helper function makes it easy to do, and we can get rid of it later if we think it's not worth it given how we call the route.